lib/process-services-cloud/src/lib/task/mock/task-cloud.service.mock.ts

```diff
@@ -162,4 +162,10 @@ export class TaskCloudServiceMock implements TaskCloudServiceInterface {
             return throwError('AppName/TaskId not configured');
         }
     }
+
+    updateColumnOrderPreferences(_appName: string, _columnsOrder: string[]): void {}
```

Comment:
Shouldn't this return something as it is into the mock?

Reply:
since preferences services are in components right now, this is now needed, removed